specutils/io/default_loaders/generic_ecsv_reader.py

```diff
@@ -6,21 +6,33 @@
 from astropy.table import Table
 from ..generic_spectrum_from_table import generic_spectrum_from_table
 
+
 def identify_ecsv(origin, *args, **kwargs):
-# check if file can be opened with this reader
-# args[0] = filename
+    """Check if it's an ECSV file."""
     return (isinstance(args[0], six.string_types) and
-    # check if file is .ecsv (would be nice to validate the header too)
-        os.path.splitext(args[0].lower())[1] == '.ecsv'
-        )
+            os.path.splitext(args[0].lower())[1] == '.ecsv')
+
 
 @data_loader("generic-ecsv", identifier=identify_ecsv, dtype=Spectrum1D)
 def generic_ecsv(file_name, **kwargs):
     """
     Read a spectrum from an ECSV file, using generic_spectrum_from_table_loader()
     to try to figure out which column is which.
+
+    Parameters
+    ----------
+    file_name: str
+        The path to the ECSV file
+
+    Returns
+    -------
+    data: Spectrum1D
+        The spectrum that is represented by the data in this table.
     """
-    logging.info("Spectrum file looks like generic-ecsv")
+    logging.info("Spectrum file looks like 'generic-ecsv'.")
```

Comment:
shouldn't this be "ECSV" instead of "generic-ecsv" given that the other readers have been renamed something like "ASCII"?

Reply:
I'm not sure when these logging statements go popularized, but I don't think we should be including them in the loaders.

Reply:
Logging statements removed.